politeiawww/backend.go

```diff
@@ -3048,6 +3034,17 @@ func (b *backend) ProcessEditProposal(user *database.User, ep www.EditProposal)
 	}, nil
 }
 
+// ProcessProposalStats returns the counting of proposals aggrouped by each proposal status
+func (b *backend) ProcessProposalsStats() www.ProposalsStatsReply {
+	ps := getProposalsStats()
+	return www.ProposalsStatsReply{
```

Comment:
Why not just reference the variables defined in `inventory.go` directly, rather than convert from a struct created in `inventory.go`?

Reply:
My first intention here is to only allow other parts of the code access the inventory through its API instead of directly accessing its values. The second point is to make the inventory API cleaner and more readable, so if any other parts of the code need to access that information its already encapsulated into a single struct.